tests/ui_tests/app/page/common/configuration/page.py

```diff
@@ -92,11 +91,21 @@ def compare_versions(self, compare_with: str, base_compare_version: Optional[str
             # to hide select panel so it won't block other actions
             self.find_element(self.locators.compare_to_select).send_keys(Keys.ESCAPE)
 
-    def click_on_advanced(self):
+    @allure.step('Click on advanced button')
+    def click_on_advanced(self, is_rows_changes=False):
         """Click on advanced button and wait rows changed"""
-        with self.wait_rows_change():
+        if is_rows_changes:
```

Comment:
this flag is really needed? it seems unused

Reply:
yep, removed